vitest.config.mjs

```diff
@@ -265,7 +265,7 @@ export default defineConfig({
 		},
 		setupFiles: ['./vitest.setup.js'],
 		globals: true,
```

Comment:
One day I hope we can get rid of globals as well 😅

Reply:
+1